Framework/Algorithms/src/SpectrumAlgorithm.cpp

```diff
@@ -0,0 +1,52 @@
+#include "MantidAlgorithms/SpectrumAlgorithm.h"
+
+#include "MantidKernel/ArrayProperty.h"
+#include "MantidKernel/BoundedValidator.h"
+#include "MantidAPI/MatrixWorkspace.h"
+
+using namespace Mantid;
+using namespace Algorithms;
+
```

Comment:
It's good to have this as common code as the naming of these things being inconsistent is constant pain.